newrelic/build_test.go

```diff
@@ -286,4 +286,27 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		Expect(result.BOM.Entries[0].Name).To(Equal("new-relic-php"))
 		Expect(result.BOM.Entries[1].Name).To(Equal("helper"))
 	})
+
+	it("contributes Python agent API >= 0.7", func() {
+		ctx.Plan.Entries = append(ctx.Plan.Entries, libcnb.BuildpackPlanEntry{Name: "new-relic-python"})
+		ctx.Buildpack.Metadata = map[string]interface{}{
+			"dependencies": []map[string]interface{}{
+				{
+					"id":      "new-relic-python",
+					"version": "8.1.0",
+					"stacks":  []interface{}{"test-stack-id"},
+				},
+			},
+		}
```

Comment:
```suggestion
```

Not necessary unless the buildpack will be downloading/installing/extracting files.

Reply:
Done